gke-disk-image-builder/README.md

```diff
@@ -44,6 +44,7 @@ Flag                | Required | Default | Description
 *--project-name*    | Yes      | nil     | Name of a gcp project where the script will be run
 *--image-name*      | Yes      | nil     | Name of the image that will be generated
 *--image-family-name* | No | secondary-disk-image | Name of the image family associated with the created disk image
+*--image-labels* | No | nil | Labels tagged to the disk image. This flag can be specified multiple times. The accepted format is `--image-labels=key=val`.
```

Comment:
This is great. Thanks a lot for the PR! Would you mind adding example for this flag as well? This would help people to try it and make it easier for them to understand how it works a bit better.

Reply:
Sounds good. I added an example.